src/devices/sonoff.ts

```diff
@@ -61,6 +61,13 @@ const sonoffExtend = {
                 deviceWorkMode: {ID: 0x0018, type: Zcl.DataType.UINT8},
                 detachRelayMode2: {ID: 0x0019, type: Zcl.DataType.BITMAP8},
                 lackWaterCloseValveTimeout: {ID: 0x5011, type: Zcl.DataType.UINT16},
+                ACCurrentCurrentValue: {ID: 0x7004, type: Zcl.DataType.UINT32},
```

Comment:
```suggestion
                acCurrentCurrentValue: {ID: 0x7004, type: Zcl.DataType.UINT32},
```

Make sure that the others also start with lower case